### ISSUE TITLE
Download buttons not working in Mutations tab in IE

### ISSUE THREAD

**Issue by marianc000** [cBioPortal/cbioportal#3300]
Hello authors

I would like to indicate two minor issues in IE support. 
The button to download pdf does not work in IE. The script tries to post an undefined value.
<img width="545" alt="pdf ie" src="https://user-images.githubusercontent.com/20133496/32218007-ea2adddc-be29-11e7-9f8c-ee1c79768575.PNG"> 

Also some IEs with some particular enterprise settings do not understand malformed html pages like that one or FAQ, News, About Us where `<script>` tags preceed `<head>` tag. 

Thank you 
Best regards,
Marian

**Comment by jjgao** [comment]
@marianc000 would you please let us know the version of your IE?

**Comment by marianc000** [comment]
Hello

I use IE 11
![image](https://user-images.githubusercontent.com/20133496/33544491-3e57ed48-d8db-11e7-82c9-2a0223b84919.png)

In which the checkbox "Display intranet sites in compatibility mode" is selected. 
![image](https://user-images.githubusercontent.com/20133496/33545075-5093089c-d8dd-11e7-8d22-801cbad07816.png)


I installed cBioPortal here in the local network. All the malformed pages that I listed above are displayed in the compatibility mode with hardly anything displayed. When I moved HEAD in front of SCRIPT tags, the pages are displayed as they are displayed in your site when it works. 

Additionally, I have found more incompatibilities with IE11. Most are due to the used javascript methods that are not implemented in ie11, for example arrays findIndex.
